Source/App/EbAppConfig.c

```diff
@@ -49,7 +49,6 @@
 #define TIER_TOKEN                      "-tier"
 #define LEVEL_TOKEN                     "-level" 
 #define LATENCY_MODE                    "-latency-mode" // no Eval
-#define INPUT_BUF_TOKEN                 "-inbuf"
```

Comment:
if -inbuf is removed and not supported, please update the sample.cfg and user guide.